test/test-utils.ts

```diff
@@ -40,6 +41,11 @@ const converters = {
     extension: ".yml",
     converter: curlconverter.toAnsible,
   },
+  cfml: {
+    name: "ColdFusion",
+    extension: ".cfm",
```

Comment:
You have .cfm here but the test files are .js

Reply:
I will update them. I chose js as the extension just due to the color highlighting working better for the script syntax.